app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<% provide :title, t(".subtitle") %>
+<h1> <%= t ".subtitle" %> </h1>
+
+<div class = "row" >
+  <div class = "col-md-6 col-md-offset-3" >
+    <%= form_for :session, url: login_path do |f| %>
```

Comment:
form_for, form_tag, form_with khác gì nhau em? Form này sẽ chuyển lên server params như thế nào?

Reply:
form_for: tạo form có đối tượng
form_tag: tạo form không có đối tượng 
form_with: có hoặc không có đối tượng đều được

form_for: :session sẽ thêm vào thuộc tính for và name của label và input còn url là action điều hướng đến controller sessions và action create